src/partials/advantages.html

```diff
@@ -0,0 +1,22 @@
+<div class="advantages container">
+  <ul class="advantages__list">
+    <li class="advantages__item list">
```

Comment:
list can be put once on <ul>
`ul class="advantages__list `list"`
  `li` class="advantages__item">`